packages/lib/modules/fee-managers/useFeeManager.ts

```diff
@@ -0,0 +1,26 @@
+import { useFeeManagers } from './FeeManagersProvider'
+import { getChainId } from '@repo/lib/config/app.config'
+import { Pool } from '../pool/pool.types'
+
+export function useFeeManager(pool: Pool) {
```

Comment:
Don't know anything about the fee manager topic. Can it only be associated with a chain or does specific pools can have their own fee manager?

Reply:
pools can optionally have their own swap fee manager, it is not chain specific